### ISSUE TITLE
Los Angeles Criminal Sentencing Project: Update alt text for Los Angeles Criminal Sentencing Project image to adhere to WCAG

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#3952]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Los Angeles Criminal Sentencing Project` image so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/criminal-sentencing.md`:

**From:**
```
Gavel
```

**To:**
```
Los Angeles Criminal Sentencing Project
```

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- Los Angeles Criminal Sentencing Project: https://www.hackforla.org/projects/criminal-sentencing

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/criminal-sentencing.md`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_projects/criminal-sentencing.md
- This issue is part of epic #3866 

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by jdingeman** [comment]
Hello @mademarc, this issue is in the New Issue Approval column and has the Ready For Milestone label, so it is not yet ready to be worked on. Thank you for your interest in taking on this issue.

**Comment by angieyan** [comment]
Availability: Feb 24 8-9pm
ETA: Feb 26, 2023
